src/pages/Checkout/CheckoutPage.vue

```diff
@@ -163,19 +163,22 @@
 		</div>
 		<div v-if="emptyBasket" class="empty-basket">
 			<div class="row display-align text-center">
-				<h2 class="empty-basket-heading">Your basket is empty!</h2>
-				<p>But we'd love to help you change that! Please consider
-				supporting one of the borrowers below, or
-					<a href="/lend-by-category">browse all loans</a>.
-				</p>
+				<div class="columns small-12">
+					<h2 class="empty-basket-heading">Your basket is empty!</h2>
+					<p>But we'd love to help you change that! Please consider
+					supporting one of the borrowers below, or
+						<a href="/lend-by-category">browse all loans</a>.
+					</p>
+				</div>
 			</div>
 
-			<random-loan-selector
-				@updating-totals="setUpdatingTotals"
-				@refreshtotals="refreshTotals"
-			/>
-
-			<loading-overlay v-if="updatingTotals" id="updating-overlay" class="updating-totals-overlay" />
+			<div class="empty-basket-loans">
```

Comment:
new container to isolate loader over our loan array only, prevents the loader background from covering up the empty basket copy.